python/pyproject.toml

```diff
@@ -17,7 +17,7 @@
 
 [tool.poetry]
 name = "pyiceberg"
-version = "0.1.0.dev0"
+version = "0.1.0"
```

Comment:
I'm not really familiar with how Poetry does versioning, but if we push this to master, doesn't that make the baseline pinned to `0.1.0` for all build artifacts until it's updated?  On the java side, I think we don't explicitly checkin a release version number.  Feels like we should only have a release version for the actual release build.

Reply:
We have a SNAPSHOT version for Java, right? My idea was to set it to `0.1.0` so we can release the code-base as is, and we don't have to update versions after the RC is accepted. After the release, I'll bump it to `0.2.0` on master.